### DISCUSSION TITLE
Any ways to make the computers require energy?

### DISCUSSION THREAD

**Discussion by LadyAelita** [cc-tweaked/CC-Tweaked#1709] [Q&A]
I'm making a tech modpach and thinking about incorporating this mod, and it just feels kinda glaring to me to have computers _just_ turn on, while every other machine (from other mods) needs to be connected to a power source. Any ideas on how to go about this?

**Comment by SquidDev** [comment]
I'm afraid not, no! This is an intentional choice - there's [a recent issue](https://github.com/cc-tweaked/CC-Tweaked/issues/1690) which goes into a little detail about why.
